### PR TITLE
chore(actions): create manual action to publish novo elements 

### PR DESCRIPTION
**Pull request by charlesabarnes**
## **Description**

Create an action that publishes Novo elements to NPM automatically.  This is run manually from the actions tab.

### CODE REVIEW

File: .github/workflows/releaseNovoElements.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Release Novo Elements
+on:
+  workflow_dispatch:
+    inputs:
+    type:
+      description: 'Update Type (patch, major, minor)'
```
Comment (ndickerson) [inline .github/workflows/releaseNovoElements.yml:6]: Not sure exactly how this gets consumed, but just for standardization, should this be:

`major, minor, patch`?

Comment (charlesabarnes) [reply .github/workflows/releaseNovoElements.yml]: That bit is just a hint for for the options available in the action.   What is input gets used by `npm version` to bump the version number in the package.json and create the tag.   Updated the order, thank you

**Review by amrutha-rajiv — APPROVED** [review]

**Review by bvkimball — CHANGES_REQUESTED** [review]
Don't we use standard-version to version novo-elements?

**Comment by charlesabarnes** [comment]
> Don't we use standard-version to version novo-elements?
@bvkimball 

You're right, I simplified the script by just changing it to run `npm run release:elements`.  The postscript will handle publishing and everything

**Review by bvkimball — APPROVED** [review]

**Pull request merged**